README.md

```diff
@@ -303,6 +303,93 @@ The credit card charge result may contains `redirect_url` for 3DS authentication
 For full example on Credit Card 3DS transaction refer to:
 - [Flask App examples](/examples/flask_app) that implement Snap & Core Api
 
+### 2.2.D Subscription API
+You can see some Subscription API examples [here](examples/subscription), [Subscription API Docs](https://api-docs.midtrans.com/#subscription-api)
+
+```python
+# Create Subscription API instance
+subscription = midtransclient.CoreApi(
+    is_production=False,
+    server_key='YOUR_SERVER_KEY',
+    client_key='YOUR_CLIENT_KEY'
+)
+# Build parameter
+param = {
```

Comment:
avoid ambiguous term like `build`, build can mean to build/compile source code into a final file. Change to `prepare parameter`